test/classes/local-file-location/absolute.test.js

```diff
@@ -1,4 +1,5 @@
 const tap = require('tap');
+const { sep } = require('path');
```

Comment:
This went unused
```suggestion
```

Reply:
https://github.com/eik-lib/rollup-plugin.git